hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -42,7 +42,7 @@
   private static final String BASE_PATH_PROP = "hoodie.base.path";
   private static final String AVRO_SCHEMA = "hoodie.avro.schema";
   public static final String TABLE_NAME = "hoodie.table.name";
-  private static final String DEFAULT_PARALLELISM = "200";
+  private static final String DEFAULT_PARALLELISM = "1500";
```

Comment:
Is this intentional ?

Reply:
yes. 200 is too small since spark partitions need to be less than 2GB and we 'd like to be able to do 500GB upserts in a stable way out of box.